recipe_scrapers/simplycookit.py

```diff
@@ -25,3 +28,30 @@ def instructions(self):
             instructions.append(normalize_string(li.get_text()))
 
         return "\n".join(instructions)
+
+    def ingredient_groups(self):
+        ingredient_groups = []
+        current_group = None
+
+        for li in self.soup.find("ul", {"class": "recipe_ingredients"}).findAll("li"):
+            h3 = li.find("h3")
+            if h3:
+                if current_group:
+                    ingredient_groups.append(current_group)
+                current_group = IngredientGroup(
+                    ingredients=[], purpose=normalize_string(h3.get_text())
+                )
```

Comment:
Nitpick: maybe rename the variable to indicate what kind of content it is, rather than what kind of HTML is used to represent it?

```suggestion
            group_heading = li.find("h3")
            if group_heading:
                if current_group:
                    ingredient_groups.append(current_group)
                current_group = IngredientGroup(
                    ingredients=[], purpose=normalize_string(group_heading.get_text())
                )
```